### ISSUE TITLE
inconsistent sort-import behavior

### ISSUE THREAD

**Issue by itajaja** [eslint/eslint#7943]
The behavior of sort-import is the following:

```js
// this is correct
import A from './z';
import Z from './a';
// this is wrong
import Z from './a';
import A from './z';
// this is correct
import { Z } from './a';
import { A } from './z';
// this is wrong
import { A } from './z';
import { Z } from './a';
```

the point here is that if it's a single import the sorting is based on the value, and if it's a multiple import the sorting is based on the path. This seems really odd, wouldn't it be more consistent if the sorting was always based on the path?

**Comment by itajaja** [comment]
oops, my bad
